ern-container-gen-android/src/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeReactContainer.java

```diff
@@ -27,6 +27,9 @@
 import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.SafeActivityStarter;
+{{#RN_VERSION_GTE_60_1}}
+import com.facebook.soloader.SoLoader;
```

Comment:
Any chance you could move this down to keep it in alphabetical order?

Otherwise
`com.facebook.soloader.SoLoader`
would be placed _above_
`com.facebook.react.modules.network.OkHttpClientFactory`

and could result in unnecessary reordering/line changes when optimizing imports.

Reply:
Done, but there are plenty of other imports in this file that are not in lexical order, so cleaning up the rest of the imports (not related to this PR) should be done through a separate PR.

It might also be very difficult for some of them because of the mustache templating that is grouping some imports based on some conditionals. 

for ex :
```
{{#conditionalA}}
com.a.b.c
com.e.f.g
{{/conditionalA}}
{{#conditionalB}}
com.c.d.e
{{/conditionalB}}
```

If both conditionals resolves to `true`, then the ordering of imports will be wrong, unless we make the mustache templating more complex. Need to find a right balance here, but I would rather favor the readability of the mustache templates rather than proper ordering given that these generated code is not really generally meant to be opened directly in Android Studio (apart from debugging purposes mostly done by our team).

Reply:
Yeah, definitely agree, it should be done in a separate PR 👍